aiida/cmdline/commands/cmd_devel.py

```diff
@@ -20,26 +22,36 @@ def verdi_devel():
 
 
 @verdi_devel.command('check-load-time')
-def devel_check_load_time():
+@click.option('-v', '--verbose', is_flag=True, help='Increase verbosity')
+def devel_check_load_time(verbose):
     """Check for common indicators that slowdown `verdi`.
 
     Check for environment properties that negatively affect the responsiveness of the `verdi` command line interface.
     Known pathways that increase load time:
 
         * the database environment is loaded when it doesn't need to be
-        * the `aiida.orm` module is imported when it doesn't need to be
+        * Only expected `aiida.*` modules are imported
 
     If either of these conditions are true, the command will raise a critical error
     """
     from aiida.manage.manager import get_manager
 
+    aiida_modules = [key for key in sys.modules if key.startswith('aiida.')]
```

Comment:
just to be more clear
```suggestion
    loaded_aiida_modules = [key for key in sys.modules if key.startswith('aiida.')]
```